iron-image.html

```diff
@@ -105,9 +114,11 @@
 
   <template>
 
-    <img id="img" role="none" hidden$="[[_computeImageVisibility(sizing)]]">
-    <div id="placeholder" hidden$="[[_computePlaceholderVisibility(fade,loaded,preload)]]" class$="[[_computePlaceholderClassName(fade,loaded,preload)]]"></div>
-    <content></content>
+    <div id="sizedImgDiv" role="img" aria-label$="{{alt}}" hidden$="[[_computeBackdropHidden(sizing)]]"></div>
+    <img id="img" alt$="{{alt}}" hidden$="[[_computeImgHidden(sizing)]]">
```

Comment:
Ok, updated again. I wrote some tests too but they're really implementation dependent. Is there any way to programmatically inspect the accessibility tree?